### PR TITLE
Mikes homework

### PR DESCRIPTION
**Pull request by mikefagerness**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
             "skipFiles": [
                 "<node_internals>/**"
             ],
-            "program": "${file}.js"
+            "program": "${file}"
```
Comment (sarachnoid) [inline .vscode/launch.json]: Hi!

File: src/john/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+
+
+
+// const myCallback = () => {
+//     const myLabel = document.createElement('div')
+//     myLabel.style.background = 'red'
+//     myLabel.innerText = "Hello Friends!"
+//     document.body.appendChild(myLabel)
```
Comment (emilychiu25) [inline src/john/index.js]: sup

**Review by sarachnoid — APPROVED** [review]

**Review by emilychiu25 — APPROVED** [review]

**Comment by sarachnoid** [comment]
John likes poop.

File: .vscode/launch.json (hunk 1 above)
Comment (mikefagerness) [reply .vscode/launch.json]: Hi!

**Pull request merged**